features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
It would be better to write this as a unit spec as it's not applicable to our documentation really, also is this a breaking change in our behaviour? Some people may be relying on inverse behaviour already...